package/MDAnalysis/analysis/align.py

```diff
@@ -245,9 +245,11 @@ def rotation_matrix(a, b, weights=None):
     `R` can be used as an argument for
     :meth:`MDAnalysis.core.groups.AtomGroup.rotate` to generate a rotated
     selection, e.g. ::
-
+    >>> from MDAnalysisTests.datafiles import TPR, TRR
+    >>> A = mda.Universe('topol.tpr','traj.trr')
```

Comment:
fix

Reply:
Yeah. I am fixing this, but I am getting errors, and I don't know why, but I need time to understand before debugging

Reply:
@orbeckst, thanks, but I am currently having my semester exams 😄. So, I will try to continue after that ✌️